roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -95,6 +95,7 @@ ntp_filter_interface: false
 #   - listen xxx
 # The NTP driftfile path
 # Only takes effect when ntp_manage_config is true.
+# For ntpsec use '/var/lib/ntpsec/ntp.drift'
 ntp_driftfile: /var/lib/ntp/ntp.drift
```

Comment:
```suggestion
# Only takes effect when ntp_manage_config is true.
# Default value is `/var/lib/ntp/ntp.drift`, for ntpsec use '/var/lib/ntpsec/ntp.drift'
ntp_driftfile: >-
      {% if ntp_package == "ntpsec" -%}
      /var/lib/ntpsec/ntp.drift
      {%- else -%}
      /var/lib/ntp/ntp.drift
      {%- endif -%}
```

WDYT?

Reply:
I like it, but should we then also put a default value for when "chrony" is used as ntp_package?

I'm not 100% sure what it should be, but in their [documentation](https://chrony-project.org/doc/3.4/chrony.conf.html) (under driftfile) they have `/var/lib/chrony/drift` as example value. Do you think we can put that here?

Reply:
Driftfile is not required to be set, so there is no default value. (only example in documentation)

You can add this in the PR, but if you want to make changes, you may need to write “release note action required” about using the upgraded clusters for notice.

Reply:
Okay I didn't realize that it was optional, then I think it makes sense to stick to your suggestion here.